backend/src/main/java/com/htecgroup/skynest/util/EmailUtils.java

```diff
@@ -1,214 +1,17 @@
-        + "        \n"
-        + "      </td>\n"
-        + "    </tr>\n"
-        + "  </tbody></table>\n"
-        + "  <table role=\"presentation\" class=\"m_-6186904992287805515content\" align=\"center\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\" style=\"border-collapse:collapse;max-width:580px;width:100%!important\" width=\"100%\">\n"
-        + "    <tbody><tr>\n"
-        + "      <td width=\"10\" height=\"10\" valign=\"middle\"></td>\n"
-        + "      <td>\n"
-        + "        \n"
-        + "                <table role=\"presentation\" width=\"100%\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\" style=\"border-collapse:collapse\">\n"
-        + "                  <tbody><tr>\n"
-        + "                    <td bgcolor=\"#1D70B8\" width=\"100%\" height=\"10\"></td>\n"
-        + "                  </tr>\n"
-        + "                </tbody></table>\n"
-        + "        \n"
-        + "      </td>\n"
-        + "      <td width=\"10\" valign=\"middle\" height=\"10\"></td>\n"
-        + "    </tr>\n"
-        + "  </tbody></table>\n"
-        + "\n"
-        + "\n"
-        + "\n"
-        + "  <table role=\"presentation\" class=\"m_-6186904992287805515content\" align=\"center\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\" style=\"border-collapse:collapse;max-width:580px;width:100%!important\" width=\"100%\">\n"
-        + "    <tbody><tr>\n"
-        + "      <td height=\"30\"><br></td>\n"
-        + "    </tr>\n"
-        + "    <tr>\n"
-        + "      <td width=\"10\" valign=\"middle\"><br></td>\n"
-        + "      <td style=\"font-family:Helvetica,Arial,sans-serif;font-size:19px;line-height:1.315789474;max-width:560px\">\n"
-        + "        \n"
-        + "            <p style=\"Margin:0 0 20px 0;font-size:19px;line-height:25px;color:#0b0c0c\">Hi "
-        + email
-        + ",</p><p style=\"Margin:0 0 20px 0;font-size:19px;line-height:25px;color:#0b0c0c\"> You have requested a password reset. If this was you, continue to the link below: </p><blockquote style=\"Margin:0 0 20px 0;border-left:10px solid #b1b4b6;padding:15px 0 0.1px 15px;font-size:19px;line-height:25px\"><p style=\"Margin:0 0 20px 0;font-size:19px;line-height:25px;color:#0b0c0c\"> <a href=\""
-        + link
-        + "\">Reset Password</a> </p></blockquote>\n Link will expire in 30 minutes. <p>See you soon</p>"
-        + "        \n"
-        + "      </td>\n"
-        + "      <td width=\"10\" valign=\"middle\"><br></td>\n"
-        + "    </tr>\n"
-        + "    <tr>\n"
-        + "      <td height=\"30\"><br></td>\n"
-        + "    </tr>\n"
-        + "  </tbody></table><div class=\"yj6qo\"></div><div class=\"adL\">\n"
-        + "\n"
-        + "</div></div>";
-  }
+  public static final String EMAIL_VERIFICATION_SUBJECT = "Password reset for SkyNest";
+  public static final String PASSWORD_RESET_SUBJECT = "Password reset for SkyNest";
+  public static final String EMAIL_VERIFICATION_TEMPLATE = "emails/verification";
```

Comment:
this class is a good candidate to be extracted in enum, meaning you only need to provide the enum key, and it will give you the template and the subject. That way you will reduce the margin for error (for example use EMAIL_VERIFICATION_SUBJECT and PASSWORD_RESET_TEMPLATE as a couple in some method), and always use the correct value pairs

Reply:
Yeah, it sound convenient to couple those for easier usage